src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -16,4 +16,7 @@ public class CliSyntax {
     public static final Prefix PREFIX_NOTES = new Prefix("nt/");
     public static final Prefix PREFIX_INCOME = new Prefix("i/");
     public static final Prefix PREFIX_AGE = new Prefix("age/");
+    public static final Prefix PREFIX_VIEW = new Prefix("v/");
+    public static final Prefix PREFIX_DELETE = new Prefix("d/");
+    public static final Prefix PREFIX_ADD = new Prefix("a/");
```

Comment:
Maybe the prefixes could be slightly different? since `a/` is also used for address. 
Instead we could have `add/`, `del/` and `view/`, so they are a bit more descriptive.